addons/armory/stringtable.xml

```diff
@@ -1,6 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project name="mts">
 	<Package name="Armory">
+        <Container name="misc">
+            <Key ID="STR_mts_Armory_subCategory">
+                <Original>Armory</Original>
+                <German>Armory</German>
+            </Key>
+        </Container>
```

Comment:
Obsolete. There's already a `STR_mts_Armory_displayName` entry with same text

Reply:
STR_mts_Armory_displayName is now in use